docs/learn/learn-staking.md

```diff
@@ -574,15 +575,37 @@ users to withdraw. For in-depth understanding, check the
 
 ### Cons of Staking
 
-- Tokens will be locked for about {{ polkadot: 28 :polkadot }}{{ kusama: 7 :kusama }} days on
-  {{ polkadot: Polkadot. :polkadot }}{{ kusama: Kusama. :kusama }} No rewards will be earned during
-  the unbonding period.
-
+- Tokens will be locked for about
+  {{ polkadot: <RPC network="polkadot" path="consts.staking.bondingDuration" defaultValue={28} filter="erasToDays"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.staking.bondingDuration" defaultValue={28} filter="erasToDays"/> :kusama }}
+  days on {{ polkadot: Polkadot. :polkadot }}{{ kusama: Kusama. :kusama }} No rewards will be earned
+  during the unbonding period.
 - Possible punishment in case of the active validator found to be misbehaving (see
   [slashing](#slashing)).
 - Lack of liquidity i.e. You would not be able to use the tokens for participating in crowdloans or
   transfer them to different account etc.
 
+#### Unbonding Period Length
+
+The
+{{ polkadot: <RPC network="polkadot" path="consts.staking.bondingDuration" defaultValue={28} filter="erasToDays"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.staking.bondingDuration" defaultValue={28} filter="erasToDays"/> :kusama }}-day
+unbonding period serves a crucial role in mitigating ex post facto slashing, particularly in the
+context of guarding against long-range attacks. When a client encounters a chain finalized by
+[GRANDPA](./learn-consensus.md#finality-gadget-grandpa) that originates more than
+{{ polkadot: <RPC network="polkadot" path="consts.staking.bondingDuration" defaultValue={28} filter="erasToDays"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.staking.bondingDuration" defaultValue={28} filter="erasToDays"/> :kusama }}
+days in the past, it lacks the security of slashing protection.
+
+Essentially, this period establishes a cadence for synchronizing with the chain or acquiring a
+checkpoint within a timeframe that engenders trust. It's worth noting that while the choice of a
+{{ polkadot: <RPC network="polkadot" path="consts.staking.bondingDuration" defaultValue={28} filter="erasToDays"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.staking.bondingDuration" defaultValue={28} filter="erasToDays"/> :kusama }}-day
+period is somewhat arbitrary, it unquestionably provides a higher level of security compared to a
+shorter period.
+
+Looking ahead, there may be a Request for Proposals (RFP) for implementing a mechanism for fast
+unbonding through an unbonding queue. Importantly, this proposed change will not alter the
+{{ polkadot: <RPC network="polkadot" path="consts.staking.bondingDuration" defaultValue={28} filter="erasToDays"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.staking.bondingDuration" defaultValue={28} filter="erasToDays"/> :kusama }}-day
+period's role in maintaining security; instead, it will ensure that an excessive number of stakers
+cannot unbond too swiftly.
```

Comment:
I think this may need more clarification. We are mentioning "Fast unbonding" and, at the same time, saying that there is no alteration in the 28-day period.

Reply:
maybe @AlistairStewart can explain this better?

Reply:
I'd say any future RFP would be beyond the scope of the wiki until it is out there to reference.

Reply:
Thank you. We will remove the RFP related text.